.github/workflows/parametric-tests.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         uses: actions/checkout@v3
         with:
           repository: 'DataDog/system-tests'
-          ref: ${{ inputs.ref }}
+          ref: kylev/enable-config-tests
```

Comment:
looks like the tests from this branch are passing! I would just make sure to revert this before merging.